packages/features/ee/teams/pages/team-members-view.tsx

```diff
@@ -14,6 +16,52 @@ import MemberInvitationModal from "../components/MemberInvitationModal";
 import MemberListItem from "../components/MemberListItem";
 import TeamInviteList from "../components/TeamInviteList";
 
+type Team = RouterOutputs["viewer"]["teams"]["get"];
+
+interface MembersListProps {
+  team: Team | undefined;
+}
+
+const checkIfExist = (comp: string, query: string) =>
+  comp.toLowerCase().replace(/\s+/g, "").includes(query.toLowerCase().replace(/\s+/g, ""));
+
+function MembersList(props: MembersListProps) {
+  const { team } = props;
+  const [query, setQuery] = useState<string>("");
+
+  const members = team?.members;
+  const membersList = members
+    ? members && query === ""
+      ? members
+      : members.filter((member) => {
+          const email = member.email ? checkIfExist(member.email, query) : false;
+          const username = member.username ? checkIfExist(member.username, query) : false;
+          const name = member.name ? checkIfExist(member.name, query) : false;
+
+          return email || username || name;
+        })
+    : undefined;
+  return (
+    <div className="flex flex-col gap-y-3">
+      <TextField
+        addOnLeading={<Search className="text-subtle h-4 w-4" />}
```

Comment:
```suggestion
```

Reply:
Talked with @Jaibles lets remove the addon and just add a placeholder that says Search...